src/services/getEditsForFileRename.ts

```diff
@@ -207,13 +208,20 @@ namespace ts {
             if (result) return result;
         }
 
-        // Then failed lookups except package.json since we dont want to touch them (only included ts/js files)
-        const result = forEach(resolved.failedLookupLocations, tryChangeWithIgnoringPackageJson);
+        // Then failed lookups that are in the list of sources
+        const result = forEach(resolved.failedLookupLocations, tryChangeWithIgnoringPackageJsonExisting)
+            // Then failed lookups except package.json since we dont want to touch them (only included ts/js files)
+            || forEach(resolved.failedLookupLocations, tryChangeWithIgnoringPackageJson);
         if (result) return result;
 
         // If nothing changed, then result is resolved module file thats not updated
         return resolved.resolvedModule && { newFileName: resolved.resolvedModule.resolvedFileName, updated: false };
 
+        function tryChangeWithIgnoringPackageJsonExisting(oldFileName: string) {
+            const newFileName = oldToNew(oldFileName);
+            return newFileName && contains(sourcePaths, newFileName) ? tryChangeWithIgnoringPackageJson(oldFileName) : undefined;

```

Comment:
Here you are comparing Path with string.. You want to handle case sensistivity when doing this comparison since Path is canonical path while fileName is not

Reply:
I switched to `fileName`, which seems like a better choice anyway...